includes/specials/SpecialUndelete.php

```diff
@@ -1134,10 +1134,19 @@ private function showHistory() {
 		}
 		if( $haveFiles ) {
 			$batch = new LinkBatch();
+			/* Wikia change begin */
 			foreach ( $files as $row ) {
-				$batch->addObj( Title::makeTitleSafe( NS_USER, $row->fa_user_text ) );
-				$batch->addObj( Title::makeTitleSafe( NS_USER_TALK, $row->fa_user_text ) );
+				if ( ( new Wikia\Util\Statistics\BernoulliTrial( 0.01 ) )->shouldSample() ) {
```

Comment:
`Special:Undelete` can trigger pretty heavy traffic spikes. We should definitely keep an eye on it and release the username-related changes separately.